core/templates/pages/exploration-editor-page/statistics-tab/charts/bar-chart.component.spec.ts

```diff
@@ -51,8 +51,10 @@ describe('Bar Chart component', function() {
         draw: () => {}
       };
 
-      // @ts-ignore window.google should have properties from google
-      // third-party library according to the lint.
+      // There is a typescript error here because it expects properties
+      // matching actual window.google. We are suppressing this error because
+      // we don't need those properties for testing purposes.
+      // @ts-ignore
```

Comment:
Same thing, ts-expect-error, ditto for this sort of use case in other spec files

Reply:
done